_pages/trivia.html

```diff
@@ -11,6 +11,34 @@ <h1>{{ page.heading }}</h1>
 </header>
 
 <div class="content subpage-content">
+  <section class="section-pad">
+    <div class="row column">
+      <h2>Schedule</h2>
+      <ul>
+        <li>When: <a href="https://time.is/0700PM_18_October_2023_in_Durham,_United_States" target="_blank">Wednesday Oct 18, 7pm EDT</a></li>
+        <li>Where: #trivia on Slack & optionally Zoom</li>
+        <li>Sign-up: See #trivia channel on Slack for Google Sheet</li>
+      </ul>
+      <h2>Rules</h2>
+      <ul>
+        <li>The DjangoCon US Code of Conduct is in effect</li>
+        <li>Don't look things up on the internet</li>
+        <li>DM your answers directly to Tim Schilling, do not post them publicly</li>
+      </ul>
+      <h2>Logistics</h2>
+      <ul>
+        <li>Teams should be limited to five people</li>
+        <li>3 rounds with 5 questions each</li>
+        <li>The questions for a round will be posted all at once.</li>
```

Comment:
```suggestion
        <li>The questions for a round will be posted all at once</li>
```

Consistency: either use periods on all bullets or none of them. Since this is the only one with a bullet, let's go that way.